tox.ini

```diff
@@ -26,6 +26,10 @@ deps =
     django18: Django>=1.8,<1.9
     django19: Django>=1.9,<1.10
     django110: Django>=1.10,<1.11
+    djangorestframework32: djangorestframework>=3.2.3,<3.3.0
```

Comment:
I have to admit I am able to understand what is going on with the version conflict solution, but I'm not clear on the specifics.  I will try to find someone who can verify these specific changes -- @cpennington would you be able to help us here?  If not, can you recommend someone?